app/javascript/components/TripFormModal/TripFormInfo.js

```diff
@@ -167,6 +170,7 @@ const TripFormInfo = () => {
                 isMulti
                 placeholder="Select from the list or create new"
                 className={styles.selectInput}
+                aria-labelledby="categories"
```

Comment:
Instead of adding an `id` for the label and mapping it to the input with `aria-labelledby`, we can use the `inputId` prop (https://react-select.com/props#select-props) which will work with the existing `for` attribute on the label.
```
 inputId={FIELD_NAMES.CATEGORIES}
```

Reply:
See first comment.